lib/services/role.go

```diff
@@ -862,6 +862,16 @@ func NewRoleSet(roles ...types.Role) RoleSet {
 // RoleSet is a set of roles that implements access control functionality
 type RoleSet []types.Role
 
+func (r RoleSet) HasModeratedSessionsPolicies() bool {
+	for _, role := range r {
+		if len(role.GetSessionRequirePolicies()) > 0 {
```

Comment:
It seems there's a `Kind` property within the return of `GetSessionRequirePolicies`
Should we check against `K8S`?

Reply:
yup